cpp/src/utilities/cython.cpp

```diff
@@ -299,6 +299,77 @@ std::pair<size_t, weight_t> call_louvain(raft::handle_t const& handle,
   return results;
 }
 
+// Wrapper for calling Pagerank through a graph container
+template <typename vertex_t, typename weight_t>
+void call_pagerank(raft::handle_t const& handle,
+                   graph_container_t const& graph_container,
+                   vertex_t* identifiers,
+                   weight_t* p_pagerank,
+                   vertex_t personalization_subset_size,
+                   vertex_t* personalization_subset,
+                   weight_t* personalization_values,
+                   double alpha,
+                   double tolerance,
+                   int64_t max_iter,
+                   bool has_guess)
+{
+  if (graph_container.graph_ptr_type == graphTypeEnum::graph_t_float_mg) {
+    pagerank(handle,
+             graph_container.graph_ptr_union.graph_t_float_mg_transposed_ptr->view(),
+             (float*)nullptr,
```

Comment:
Just nitpicking... static_cast<float*>(nullptr) will be better, same for the code below this. Better avoid old C style casting